test/e2e/providers/vsphere/vsphere.go

```diff
@@ -10,16 +10,15 @@ import (
 	"github.com/pkg/errors"
 
 	config "github.com/openshift/api/config/v1"
-	mapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
-	vsphere "github.com/openshift/machine-api-operator/pkg/apis/vsphereprovider/v1beta1"
+	mapi "github.com/openshift/api/machine/v1beta1"
 	core "k8s.io/api/core/v1"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
 	"github.com/openshift/windows-machine-config-operator/test/e2e/clusterinfo"
 )
 
-const defaultCredentialsSecretName = "vsphere-cloud-credentials"
+const defaultCredentialsSecretName = "mapi-cloud-credentials"
```

Comment:
The vSphere e2e test is [failing](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/788/pull-ci-openshift-windows-machine-config-operator-master-vsphere-e2e-operator/1455974928560951296#1:build-log.txt%3A93) because of this.